helm/event-exporter-app/templates/configmap.yaml

```diff
@@ -1,183 +1,176 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: event-exporter-app
-  namespace: giantswarm
+  name: {{ include "event-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "event-exporter.labels" . | nindent 4 }}
 data:
   config.yaml: |
-    logLevel: debug
```

Comment:
Do we no longer want this?

Reply:
Depends on. I'd not use log level debug in production. If we ever need it, we can still add it for actual debugging. But until then it probably only wastes resources, assuming we're shipping logs elsewhere.